Gems/Atom/Asset/ImageProcessingAtom/Code/Source/ImageBuilderComponent.cpp

```diff
@@ -242,13 +294,26 @@ namespace ImageProcessingAtom
             if (ImageProcessingAtom::BuilderSettingManager::Instance()->DoesSupportPlatform(platformInfo.m_identifier))
             {
                 AssetBuilderSDK::JobDescriptor descriptor;
-                descriptor.m_jobKey = ext + " Atom Compile";
+                descriptor.m_jobKey = "Image Compile: " + ext;
```

Comment:
The job key is not just for display, but also allows other builders to depend on this job.  

While there's nothing necessarily wrong with giving them a different job key per extension, it means that nobody can declare a dependency on just "Atom Image Compile" for any random source image -  they have to do extra work to determine the type of file by extension and synthesize the same string.

It also means that when stats are printed, each different type of image compile will have its own stat.  

The job key also uniquely idenfies the job in the database - if you change this, you invalidate the job and it will be rerun next time.

Reply:
Good to know. 
I believe the image job key wasn't used as dependency from others. I will leave it as is. We may change it if things changed

Reply:
I am expecting to add an OrderOnce job dependency between .material and image files at some point. It would be good to remove the extension. If it isn't removed now, I'll bring this up again when we get to that point.